content/episode/devney-hamilton-sassafras-tech.md

```diff
@@ -0,0 +1,351 @@
++++
+title = "Devney Hamilton on Sassafras Tech Collective"
+date = 2021-03-18T15:08:22-07:00
+description = "Devney Hamilton talks about Sassfras Tech Collective, Sassafras' internal structures, and the importance of prefigurative processes."
+tags = []
+draft = false
+podcast_file = "https://ia601506.us.archive.org/22/items/devney-hamilton-on-sassafras-tech-collective/Devney%20Hamilton%20on%20Sassafras%20Tech%20Collective.mp3"
+podcast_duration = "47:44"
+podcast_bytes = "45933627"
+episode_image = "/img/devney-hamilton-sassfras.jpg"
++++
+
+## Episode Notes
+
+Sassafras encourages people of color and/or with disabilities and/or trans or gender-non-conforming, or otherwise targeted with oppression in tech to contact them at [info@sassafras.coop](mailto:info@sassafras.coop).
+
+Ways to stay in touch with Sassafras:
+* Website: [sassafras.coop](https://www.sassafras.coop/)
+* Twitter: [@sassafrastech](https://twitter.com/sassafrastech)
+* Blog: [blog.sassafras.coop](https://blog.sassafras.coop/)
+
+Organizations mentioned:
+* [Atomic Object](https://atomicobject.com/), a Midwest software consultancy, employee-owned B corporation
+* [Loomio](https://help.loomio.org/en/about/), a software product co-op.
+* [Allied Media Conference](https://amc.alliedmedia.org/)
+* [INCITE! Women of Color Against Violence](https://incite-national.org/)
+* [Design Action Collective](https://designaction.org/)
+
+Places to learn about co-op work opportunities:
+* [USFWC Tech Co-op Peer Network](https://info.usworker.coop/civicrm/event/info?reset=1&id=429)
+* [Women in Tech Slack](https://witchat.github.io/)
+
+Co-ops that support Sassafras' work:
+* [AORTA](https://aorta.coop/), Anti-Oppression Resource and Training Alliance
+* [Open Bookkeeping](http://www.openbookkeeping.com/)
+* [Key Figures](https://www.keyfigures.coop/)
+* [story2designs](https://story2designs.com/)
+* [Design Justice Network](https://designjustice.org/)
+
+Organizations that help fund cooperatives:
+* [The Working World](https://www.theworkingworld.org/)
+* [Seed Commons](https://seedcommons.org/)
+
+Organizations that provide resources for starting coops:
+* [US Federation of Worker Coops](https://www.usworker.coop/)
+* [Sustainable Economies Law Center](https://www.theselc.org/)
+* [Democracy at Work Institute](https://institute.coop/)
+
+Books and reading recommended:
+* [Prefigurative Design as a Method for Research Justice](https://dl.acm.org/doi/pdf/10.1145/3359302), research paper by Mariam Asad
+* [Collective Courage: A History of African American Cooperative Economic Thought and Practice](https://www.psupress.org/books/titles/978-0-271-06216-7.html), book by Jessica Gordon Nembhard
+* [Emergent Strategy: Shaping Change, Changing Worlds](https://www.akpress.org/emergentstrategy.html), book by adrienne maree brown
+
+## Transcript
+
```

Comment:
The launch.coop conversation is all set to keep included :) thanks for double checking!